Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
```

Comment:
what's the purpose of this and how is it used?  it looks like you already added pip install toil stuff in `toil-jumpbox-test.yaml`?

Reply:
pipenv uses a Pipfile. I started a pipenv environment to try to do the development so I don't have to make many commits to work out all the build issues, but sceptre isn't working for me locally even with the admin role, so maybe I'll just remove these.